install_config/configuring_red_hat_registry.adoc

```diff
@@ -0,0 +1,127 @@
+[[install-config-configuring-red_hat_registry]]
+= Accessing and Configuring the Red Hat Registry
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[auth-enabled-registry]]
+== Authentication Enabled Red Hat Registry
+
+include::architecture/topics/auth_enabled_registry.adoc[leveloffset=+1]
+
+
+[[create-authentication-tokens]]
+=== Creating Service Accounts and Authentication Tokens
+
+You must create tokens if your organization manages shared accounts.
+Administrators can create, view, and delete all tokens associated with an
+organization.
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+
+//[[Procedure]]
+*Procedure*
+
+To create a token in order complete a `docker login`:
+
+. Navigate to `registry.redhat.io`.
+. Log in with your RHN username and password.
+. Accept terms when prompted.
+.. If you are not immediately prompted to accept terms, you will be prompted
+when proceeding with the following steps.
+. From the *Registry Service Accounts* page, click *Create Service Account*
+.. Provide a name for the service account. It will be prepended with a random string.
+.. Enter a description.
+.. Click create.
+. Navigate back to your Service Accounts.
+. Click the Service Account you created.
+. Copy the username, including the prepended string.
+. Copy the token.
+
+[[using-service-accounts]]
+=== Using Service Accounts
+
+Once you have created your service accounts and generated tokens, you can perform
+additional tasks.
+
+[NOTE]
+====
+This section provides the manual steps, which can be automatically performed
+during installation.
+====
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+* Service account
+* Service account token
+
+
+From your link:https://access.redhat.com/terms-based-registry/[*Registry Service
+Accounts*] page, click on your account name. From there, you can perform the
+following tasks:
+
+* Under the *Token Information* tab, you can view your username (the name you provided
+ prepended with a random string) and password (token). From this tab, you can
+ regenerate your token.
+* Under the *OpenShift Secret* tab, you can:
+.. Download the secret by clicking the link in the tab.
+.. Submit the secret to the cluster, for example:
++
+----
+# oc create -f <account-name>-secret.yml --namespace=openshift
+----
++
+.. Update your Kubernetes configuration by adding a reference to the secret to
+your Kubernetes pod configuration with an `imagePullSecrets` field, for example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: somepod
+  namespace: all
+  spec:
+    containers:
+      - name: web
+      image: registry.redhat.io/REPONAME
+
+    imagePullSecrets:
+      - name: <numerical-string-account-name>-pull-secret
+----
++
```

Comment:
Can we break these out into separate steps instead of nesting actions under the example?